.docs/README.md

```diff
@@ -89,16 +83,15 @@ public $recurrentPaymentService;
 public $preAuthorizedPaymentService;
 ```
 
-### Před platbou
+### Before payment
+
+Before payment, you need to create a form with the corresponding payment buttons. Each payment channel is represented by one button. You can add to the form  Buttons simply add via **Binder** with the `bindPaymentButtons ()` method:
```

Comment:
Before payment, you need to create a form with the corresponding payment buttons. Each payment channel is represented by one button. You can add buttons to the form simply  via **Binder** with the `bindPaymentButtons ()` method: